src/MetricRenderer.php

```diff
@@ -17,6 +17,6 @@ public function render(Metric $metric): string
             }, array_keys($metric->getLabels()), $metric->getLabels())));
         }
 
-        return sprintf("%s%s %s\n", $metric->getName(), $labels, $metric->getValue());
+        return sprintf("%s%s %s \n", $metric->getName(), $labels, $metric->getValue());
```

Comment:
Is this new space necessary?

Reply:
To be on the safe side, yes. I have now seen this with several other packages, even the Prometheus PHP library. This should make sure that Prometheus never has problems with the "value", especially with integers or floats.